parcels/grid.py

```diff
@@ -3,7 +3,9 @@
 from enum import IntEnum
 
 import numpy as np
+import numpy.typing as npt
```

Comment:
Why don't we use `npt` in the other files too? This is the first I've now come across during this review?

Reply:
yes, can do